tests/test_cmdline.py

```diff
@@ -710,6 +710,25 @@ def test_check_detailed__simple_project_in_fake_venv__resolves_imports_vs_deps(
     assert returncode == 0
 
 
+def test_check_detailed__simple_project_w_2_fake_venv__resolves_imports_vs_deps(
+    fake_venv, project_with_code_and_requirements_txt
+):
+    tmp_path = project_with_code_and_requirements_txt(
+        imports=["some_module", "other_module", "requests"],
+        declares=["pandas", "some_module"],
+    )
+    venv_dir1, _ = fake_venv({"some_module": {"some_module"}})
+    venv_dir2, _ = fake_venv({"pandas": {"requests"}, "some_module": {"other_module"}})
```

Comment:
Nitpick alert: I found the package names and their exposed import names confusing to read. Is there a reason to not:
- either use actual package names even if this is a fake env?
- or use completely made up names for all packages and imports?

Seeing `pandas` expose `requests` is a tiny bit jarring.

Reply:
Agreed. I believe what I was thinking about when I wrote this was to demonstrate (along with grabbing packages from multiple pyenvs) that you could have imports and declared deps that don't seem to line up at all, but that - given the right `Package` object - actually lines up perfectly in the end. However, using `pandas` and `requests` to demonstrate this is just stupid on my part. 🤪 Will fix!